bin/bios-baseline.sh

```diff
@@ -1,27 +1,27 @@
 #!/bin/bash
```

Comment:
This file has too many to fix, and the ilorest stuff is _really_ temperamental.. we can pass this one without fixing it up.